.github/workflows/auto-deploy-staging.yml

```diff
@@ -0,0 +1,69 @@
+name: Auto deploy staging AWS
+env: 
+  NODE_VERSION: '12.x'
```

Comment:
when can we get rid of this node version requirement?

Reply:
you want it updated to 14? I think we always need to have it

Reply:
I'd rather just let it be whatever github-actions uses by default, unless we have some compatibility reason to hold it back.

Reply:
it would also be cleaner if we could define it once and include it in all the other scripts, but I'm not sure how to do that with gh-actions/yaml

Reply:
i'm pretty sure you always have to define what node version you're using. Also our local dev env dont auto update to the next node or anything so I think it's good to know it's the same rather than have the remote sometime using a newer version